docs/gl_objects/issues.rst

```diff
@@ -225,3 +225,51 @@ Link issue ``i1`` to issue ``i2``::
 Delete a link::
 
     i1.links.delete(issue_link_id)
+
+Issues statistics
+=========================
+
+Reference
+---------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.IssuesStatistics`
+  + :class:`gitlab.v4.objects.IssuesStatisticsManager`
+  + :attr:`gitlab.issuesstatistics`
+  + :class:`gitlab.v4.objects.GroupIssuesStatistics`
+  + :class:`gitlab.v4.objects.GroupIssuesStatisticsManager`
+  + :attr:`gitlab.v4.objects.Group.issuesstatistics`
+  + :class:`gitlab.v4.objects.ProjectIssuesStatistics`
+  + :class:`gitlab.v4.objects.ProjectIssuesStatisticsManager`
+  + :attr:`gitlab.v4.objects.Project.issuesstatistics`
+
+
+* GitLab API: https://docs.gitlab.com/ce/api/issues_statistics.htm
+
+Examples
+---------
+
+Get statistics of all issues the user has access to::
```

Comment:
The docs say that it returns by default the issues created by the current user.

https://docs.gitlab.com/ee/api/issues_statistics.html#get-issues-statistics

```
By default it returns only issues created by the current user.
```

Reply:
Thanks, done!